_test/test_unique_objects_container/test_unique_references.m

```diff
@@ -467,6 +468,7 @@ function throw1()
             assertEqual(lw,'HERBERT:unique_references_container:incomplete_setup');
             assertEqual(lwn, 'baseclass not initialised, using first assigned type');
             clear cl4b;
+            lastwarn('');
```

Comment:
What does this do?

Reply:
removes the last warning so it does not confuse the next tests